storybook/stories/API/shapes/Rectangle.mdx

```diff
@@ -0,0 +1,31 @@
+import { ArgTypes, Meta, Story, Canvas, Controls } from '@storybook/blocks';
+import * as ComponentStories from './Rectangle.stories';
+
+# Rectangle
+
+The Rectangle is a shape that is valid in the context of any SVG as well as within the `<Customized />` component.
```

Comment:
@nikolasrieble I think the other shape stories have something about needing to be within a `Customized` component. I don't think this is true in reality - they just need to be within an `svg` and have the right props sent.

Reply:
Cool! Could you please update the other stories as well, to align with this one?

Reply:
Yeah, will do